operators/src/pro/cache/tile_cache.rs

```diff
@@ -29,16 +30,425 @@ pub struct TileCacheBackend {
     // for each operator graph, we have a cache, that can efficiently be accessed
     operator_caches: HashMap<CanonicOperatorName, OperatorTileCache>,
 
-    cache_byte_size_total: usize,
-    cache_byte_size_used: usize,
-
-    landing_zone_byte_size_total: usize,
-    landing_zone_byte_size_used: usize,
+    cache_size: TileCacheBackendSize,
+    landing_zone_size: TileCacheBackendSize,
 
     // we only use the LruCache for determining the least recently used elements and evict as many entries as needed to fit the new one
     lru: LruCache<CacheEntryId, CanonicOperatorName>,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub struct TileCacheBackendSize {
```

Comment:
is this in any way specific to the tile cache? If not, I think you could name it differently and put it somewhere else, e.g. to the ByteSize trait or to a util file.

Reply:
will try to do that when building the vector cache